### PR TITLE
sys-auth/pambase: Change pam_sss handling for account stanza

### PR DESCRIPTION
**Pull request by mjg59**
Having this be effectively required isn't appropriate when we default to
having sssd be disabled.

### CODE REVIEW

**Comment by mischief** [comment]
i think you missed a `git add`.

**Comment by mjg59** [comment]
Oops. Yup.

File: sys-auth/pambase/files/pambase-20120417-sssd.patch [hunk 1]
```diff
@@ -15,7 +15,7 @@ diff -ur pambase-20140313.orig/system-auth.in pambase-20140313/system-auth.in
  #endif
  account		required	pam_unix.so DEBUG
  /* This is needed to make sure that the Kerberos skip-on-success won't cause a bad jump. */
```
Comment (marineam) [inline sys-auth/pambase/files/pambase-20120417-sssd.patch]: Is this comment still accurate?

**Comment by crawford** [comment]
`coreos.autologin`, `set_shared_password.sh`, and local `ssh` to my QEMU machine work now.

**Comment by marineam** [comment]
(comment can be fixed later if needed)

**Pull request merged**